components/widgets/inputs/users_emails_input.jsx

```diff
@@ -65,7 +67,10 @@ export default class UsersEmailsInput extends React.Component {
     }
 
     formatUserName = (user) => {
```

Comment:
Is this the same logic as the other component? If so it's an argument for reuse.